src/qibocal/protocols/characterization/flux_dependence/resonator_crosstalk.py

```diff
@@ -88,24 +89,29 @@ def _acquisition(
         type=SweeperType.OFFSET,
     )
 
-    delta_bias_range = np.arange(
-        -params.bias_width / 2, params.bias_width / 2, params.bias_step
-    )
     if params.flux_qubits is None:
         flux_qubits = list(platform.qubits.keys())
 
     else:
         flux_qubits = params.flux_qubits
-
-    bias_sweepers = [
-        Sweeper(
-            Parameter.bias,
-            delta_bias_range,
-            qubits=[platform.qubits[flux_qubit]],
-            type=SweeperType.OFFSET,
+    drive_readout_duration = sequence.duration
+    if params.flux_pulses:
+        sweepers = create_flux_pulse_sweepers(
+            params, platform, qubits, drive_readout_duration
```

Comment:
Should it be like this for the crosstalk routines?
```suggestion
        sweepers = create_flux_pulse_sweepers(
            params, platform, [platform.qubits[flux_qubit] for flux_qubit in flux_qubits], drive_readout_duration
```